### ISSUE TITLE
Create docker image and binder

### ISSUE THREAD

**Issue by bonh** [MeasureTransport/MParT#117]
A docker image can be created easily using [`jupyter-repo2docker`](https://repo2docker.readthedocs.io/en/latest/). The same tool is used by [`binder`](https://mybinder.org/). Binder allows to run jupyter notebooks interactively in the browser.

**Comment by mparno** [comment]
See start of binder extensions in the [SIAMannual22](https://github.com/MeasureTransport/SIAMannual22) repository.   After this tutorial, we should be able to build on that repository and it's binder capability to include other examples.

**Comment by bonh** [comment]
The binder stuff works, see the corresponding branch or https://mybinder.org/v2/gh/MeasureTransport/MParT/117-create-docker-image-and-binder. I am done with this issue. @mparno do you want to keep it open until we have decided what to do with the examples and binder?

**Comment by bonh** [comment]
@mparno I like your implementation of binder in SIAMannual22 better. Expect the usage of jupytext but that is very minor. So I suggest that I remove my branch corresponding to that issue and we close the issue here. Perhaps open up a discussion about what to do with the examples.

**Comment by mparno** [comment]
Hi @bonh .  Sorry for the duplication!   I think closing this issue is a good idea.   I would actually suggest we create a new repository with the same structure as `SIAMannual22` to hold the examples (maybe something like `MParT-examples`).   I'm not of the best way from including notebooks from another repo in our sphinx documentation, but I think we can figure that out.   What do you think?

**Comment by bonh** [comment]
`MParT-examples` sounds good. I do not know about sphinx and notebooks. Do we have to know before we start a new repo? If so, we should add a new issue about that.
